Remove link to post-training survey (#1794)
carpentries/instructor-training: episodes/25-wrap-up.md

::::::::::::::::::::::::::::::::::::::::::::::::::

::: instructor

> You will need to get the post-training survey link yourself and provide it in the activity below. It should follow the form `https://carpentries.typeform.com/to/cjJ9UP#slug=...../` and can be found on the website for the specific training.

Reviewer: Does the email sent to trainers not include the survey links anymore @sheraaronhurt ?

Author: @brownsarahm I'm pretty sure it still does. I can edit the update to indicate it can be found in either the email or the website, but I'll wait and see if Sher has anything to say first.

> You will need to get the post-training survey link yourself and provide it in the activity below. It should follow the form `https://carpentries.typeform.com/to/cjJ9UP#slug=...../` and can be found in an email trainers receive from the Carpentries team and on the website for the specific training.

Reviewer: I think per #1732 we are not doing the individual workshop websites anymore, @sheraaronhurt or @tobyhodges can you confirm? 

```suggestion
You will need to get the post-training survey link yourself and provide it in the activity below. It should follow the form `https://carpentries.typeform.com/to/cjJ9UP#slug=...../` and can be found in an email trainers receive from the Carpentries Core Team before the training.
```

Author: @sheraaronhurt or @tobyhodges can we have confirmation on this?

Reviewer 2: This looks right to me. Indeed Trainers are not required to create websites for individual events anymore.
